src/main/io/rc_controls.c

```diff
@@ -302,6 +302,17 @@ void processRcStickPositions(rxConfig_t *rxConfig, throttleStatus_e throttleStat
 
 }
 
+bool isModeActivationConditionPresent(modeActivationCondition_t *modeActivationConditions, boxId_e modeId)
```

Comment:
Good point. I will refactor useRcControlsConfig() to use isModeActivationConditionPresent() and add extra check for range usability.